packages/x-codemod/src/v6.0.0/pickers/rename-components-to-slots/actual-props.spec.js

```diff
@@ -0,0 +1,22 @@
+import { NextDatePicker } from '@mui/x-date-pickers';
+import { NextDateRangePicker } from '@mui/x-date-pickers-pro';
+
+<div>
+  <NextDateRangePicker
```

Comment:
Can we add another component (`TextField` for example) and check if it is renamed as well ?